Module/STIG/Functions.Checklist.ps1

```diff
@@ -57,7 +57,14 @@ function New-StigCheckList
 
     )
 
-    $manualCheckData = Invoke-Expression (Get-Content $manualCheckFile | Out-String)
+    if ('' -ne $ManualCheckFile)
+    {
+        if (-not (Test-Path -Path $ManualCheckFile))
+        {
+            throw "$($ManualCheckFile) is not a valid path to a ManualCheckFile. Provide a full valid path"
+        }
+        $manualCheckData = Invoke-Expression (Get-Content $manualCheckFile | Out-String)
```

Comment:
Invoke-Expression isn't needed here.  And lets use named parameters so it should look something like this:
$manualCheckData = Get-Content -Path $manualCheckFile | Out-String.

Is Out-String necessary?

Reply:
It's not. That wasn't code that I wrote and I understand why they did it. There is a better way and I've updated it.